crates/bevy_ui/src/ui_node.rs

```diff
@@ -866,6 +881,33 @@ impl Default for Display {
     }
 }
 
+/// Whether to show or hide overflowing items
+#[derive(Copy, Clone, PartialEq, Eq, Debug, Reflect)]
+#[reflect(Default, PartialEq)]
+#[cfg_attr(
+    feature = "serialize",
+    derive(serde::Serialize, serde::Deserialize),
+    reflect(Serialize, Deserialize)
+)]
+
+/// Which part of a Node's box length styles like width and height control
```

Comment:
nit: I think we're generally placing doc comments above attributes, and not including new lines between